pkg/metadata/host/host.go

```diff
@@ -28,6 +28,14 @@ import (
 
 const packageCachePrefix = "host"
 
+// Collect at init time
+var cpuInfo []cpu.InfoStat
+
+func init() {
+	// Collect at init to avoid COM model mayhem on windows
```

Comment:
Is this still needed, even after we've made sure importing `pythoncom` doesn't have undesirable side effects on the com initialization?

Reply:
So the reason I moved this here is because I wasn't sure I could guarantee `getCPUInfo()` (call sequence: `GetPayload()`->`getSystemStats()`-> `getCPUInfo()`) would happen before the first check run for any WMI-based check. I kept it in the init just to be safe.

Reply:
cool, thanks for the explanation, makes sense. I'm slightly concerned about calling a "complex" function in the `init`, if the function blocks indefinitely (or has other weird behaviors) it may block the initialization of the whole application... Could we move this outside of the `init` in another function and call that function explicitly from the main code of the Agent? (disregard if I'm being paranoid)